openedx/features/edly/permissions.py

```diff
@@ -0,0 +1,14 @@
+"""
+Custom permissions for edly.
+"""
+from rest_framework.permissions import BasePermission
+
+
+class CanAccessEdxAPI(BasePermission):
+    """
+    Checks if a user can access Edx API.
+    """
+
+    def has_permission(self, request, view):
```

Comment:
@burhandodhy this permission is not enough we should make sure only previliged users or only those users who belong to a particular ground should be able to enroll other users then themselves. With current permission any user can enroll anyother user as long as both belong to same site.

Reply:
@ziafazal this permission class only restrict that user of one site can't request on the other class. The class is written in a generic way so we could the same class for the other API in future.

As to restrict the enrollment @taimoor-ahmed-1 has created a separate PR https://github.com/edly-io/edx-platform/pull/300/files